common/level.go

```diff
@@ -170,10 +172,14 @@ type layer struct {
 
 func createTileset(lvl *Level, sheets []*tilesheet) map[int]*Tile {
```

Comment:
Should we somehow expose the tilesheet's tile width/height or are we OK with only having them in the tiles (in the map[int]*Tile) as now?

Reply:
Do you need the tiles exposed or can you get by with the map?

Reply:
With this modification (as is) I seem to get by (at least now :) ).